################################################################################
SHORT DESCRIPTION: 
################################################################################
Resolve OpenXT specific errors for blktap3.

################################################################################
LONG DESCRIPTION: 
################################################################################
Inclusion of blktap3 into OpenXT requires few changes to be done as below.

The wrappers in mockatests is removed because it requires libcmocka which is not
provided by any packages. This doesn't impact any functionality as it's a
testing framework. Since we are aligning with upstream, this removal shouldn't 
be an issue. However, as and when libcmocka is available it should be included.

################################################################################
CHANGELOG 
################################################################################
Author(s):
Mahantesh Salimath <salimathm@ainfosec.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
No

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/include/Makefile.am
+++ b/include/Makefile.am
@@ -19,6 +19,7 @@ blktap_HEADERS += tap-ctl.h
 blktap_HEADERS += debug.h
 blktap_HEADERS += util.h
 blktap_HEADERS += ../drivers/tapdisk-metrics-stats.h
+blktap_HEADERS += list.h
 
 noinst_HEADERS  = blktap.h
 noinst_HEADERS += compiler.h
--- a/mockatests/Makefile.am
+++ b/mockatests/Makefile.am
@@ -1,5 +1,5 @@
 # ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS  = wrappers
-SUBDIRS += drivers
+#SUBDIRS  = wrappers
+SUBDIRS = drivers
 SUBDIRS += cbt
